cmd/starter/main.go

```diff
@@ -74,11 +75,18 @@ func waitForBuild(conn *beanstalk.Conn) {
 			panic(err)
 		}
 
-		repo := string(body)
+		job, err := buildjob.FromBytes(body)
+		if err != nil {
+			// there was a job we could not parse throw it away and continue
+			_ = conn.Delete(id)
+			continue
```

Comment:
As this case should only appear after the migration towards this fix (there are some builds cycling in the queue like forever and just containing a string like it was the old version) I think this doesn't need to get logged. The only thing adding jobs into the queue is the `webhook.go` in the frontend and that one uses a format which is readable by the decoder…